CI failed in build-and-test-windows

## Bug Report

Please answer these questions before submitting your issue. Thanks!

### 1. Minimal reproduce step (Required)

run github workflow build-and-test-windows

### 2. What did you expect to see? (Required)

all tests should pass

### 3. What did you see instead (Required)

build-and-test-windows always failed since https://github.com/kcl-lang/kcl-go/commit/16ee99bd07f70a6eef2131668dd3b65c3bbe2e58. It seems that there is some problem with cgo

```
# kcl-lang.io/kcl-go/pkg/kcl_plugin/hello_plugin.test
C:\hostedtoolcache\windows\go\1.19.13\x64\pkg\tool\windows_amd64\link.exe: running gcc failed: exit status 1
C:/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/12.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:\Users\RUNNER~1\AppData\Local\Temp\go-link-425057140\000006.o: in function `_cgo_preinit_init':
\\_\_\runtime\cgo/gcc_libinit_windows.c:40: undefined reference to `__imp___iob_func'
C:/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/12.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:\Users\RUNNER~1\AppData\Local\Temp\go-link-425057140\000006.o: in function `x_cgo_notify_runtime_init_done':
\\_\_\runtime\cgo/gcc_libinit_windows.c:[105](https://github.com/kcl-lang/kcl-go/actions/runs/6609481621/job/17950015784#step:4:106): undefined reference to `__imp___iob_func'
C:/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/12.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:\Users\RUNNER~1\AppData\Local\Temp\go-link-425057140\000006.o: in function `_cgo_beginthread':
\\_\_\runtime\cgo/gcc_libinit_windows.c:149: undefined reference to `__imp___iob_func'
C:/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/12.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:\Users\RUNNER~1\AppData\Local\Temp\go-link-425057140\000007.o: in function `x_cgo_thread_start':
\\_\_\runtime\cgo/gcc_util.c:18: undefined reference to `__imp___iob_func'
collect2.exe: error: ld returned 1 exit status

# kcl-lang.io/kcl-go/pkg/kcl_plugin.test
FAIL	kcl-lang.io/kcl-go/pkg/kcl_plugin [build failed]
C:\hostedtoolcache\windows\go\1.19.13\x64\pkg\tool\windows_amd64\link.exe: running gcc failed: exit status 1
FAIL	kcl-lang.io/kcl-go/pkg/kcl_plugin/hello_plugin [build failed]
C:/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/12.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:\Users\RUNNER~1\AppData\Local\Temp\go-link-1519708659\000006.o: in function `_cgo_preinit_init':
\\_\_\runtime\cgo/gcc_libinit_windows.c:40: undefined reference to `__imp___iob_func'
?   	kcl-lang.io/kcl-go/pkg/kclvm_runtime	[no test files]
C:/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/12.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:\Users\RUNNER~1\AppData\Local\Temp\go-link-1519708659\000006.o: in function `x_cgo_notify_runtime_init_done':
\\_\_\runtime\cgo/gcc_libinit_windows.c:105: undefined reference to `__imp___iob_func'
ok  	kcl-lang.io/kcl-go/pkg/logger	0.030s [no tests to run]
C:/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/12.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:\Users\RUNNER~1\AppData\Local\Temp\go-link-1519708659\000006.o: in function `_cgo_beginthread':
?   	kcl-lang.io/kcl-go/pkg/path	[no test files]
\\_\_\runtime\cgo/gcc_libinit_windows.c:149: undefined reference to `__imp___iob_func'
ok  	kcl-lang.io/kcl-go/pkg/service	1.047s
C:/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/12.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:\Users\RUNNER~1\AppData\Local\Temp\go-link-1519708659\000007.o: in function `x_cgo_thread_start':
ok  	kcl-lang.io/kcl-go/pkg/settings	0.047s
\\_\_\runtime\cgo/gcc_util.c:18: undefined reference to `__imp___iob_func'
?   	kcl-lang.io/kcl-go/pkg/spec/gpyrpc	[no test files]
collect2.exe: error: ld returned 1 exit status
ok  	kcl-lang.io/kcl-go/pkg/tools/format	0.062s
```

--- a/.github/workflows/main_windows.yml
+++ b/.github/workflows/main_windows.yml
@@ -7,7 +7,7 @@ on:
       - "releases/*"
 jobs:
   build-and-test:
-    runs-on: windows-latest
+    runs-on: windows-2019
     steps:
       - name: Git checkout
         uses: actions/checkout@v2

